app/views/posts/show.html.erb

```diff
@@ -14,7 +14,15 @@
             <p class="each-commnet">No comments yet!</p>
         <% else %>
             <% @post.comments.each do |comment| %>
-            <p class="each-comment">@<%= comment.user.name %>: <%= comment.text %></p>
+                <div class='comment-delete'>
+                 <p class="each-comment">@<%= comment.user.name %>: <%= comment.text %></p>
+
+                 <% if can? :destroy, comment %>
+                   <%= form_with(model: comment, url: user_post_comment_path(@post.author, @post, id: comment.id), method: :delete, id: 'delete-comment-form') do |form| %>
```

Comment:
- [ ] Well done @the-faizmohammad and @ReemMohamedAbdelfatah, you did a great job by implementing the changes as suggested by your previous code reviewer, but I noticed that, when I click on a specific post to see the comments, likes, `delete` comment etc, I got an error for that specific post and the reason behind the error is the `wrong` path that you gave for the `URL`. The path should be `user_post_comments_path` not ❌ `user_post_comment_path`. 

![image](https://github.com/the-faizmohammad/BlogApp/assets/66476329/8c1f3651-a76a-483b-92ad-af654701cf1d)


- [ ] After solving the above issue, I got another issue. I saw that when I click on the delete button to remove a particular comment, it deletes the comment from that place and `renders` the comment again and again on the page, I suspect that your `delete` comment action is not working, which means you button click event only refresh the page but not deleting the comment from the database. Kindly fix the behaviour and make sure that your application should work properly and in the expected way. Remember that only the admin should be allowed to `delete` any person's comments or `posts`. Rest the `normal` login user should only be able to delete its own `comments` or `posts`.

[Screencast from 10-02-24 01:11:21 AM IST.webm](https://github.com/the-faizmohammad/BlogApp/assets/66476329/b831d50d-1fd8-4add-bcf5-d09f9fe79261)